source/Halibut/HalibutRuntimeBuilder.cs

```diff
@@ -21,7 +22,19 @@ public class HalibutRuntimeBuilder
         AsyncHalibutFeature asyncHalibutFeature = AsyncHalibutFeature.Disabled;
         Func<string, string, UnauthorizedClientConnectResponse> onUnauthorizedClientConnect;
         HalibutTimeoutsAndLimits halibutTimeoutsAndLimits;
-
+        IBaseStreamWrapper baseStreamWrapper = new NoOpBaseStreamWrapper();
+
+        /// <summary>
+        /// For testing only.
+        /// </summary>
+        /// <param name="baseStreamWrapper"></param>
+        /// <returns></returns>
+        internal HalibutRuntimeBuilder WithBaseStreamWrapper(IBaseStreamWrapper baseStreamWrapper)
```

Comment:
I don't know if BaseStreamWrapper really helps describe what it does. I think a good comment is needed here